ssd1306.h

```diff
@@ -82,6 +82,13 @@ namespace pico_ssd1306 {
         /// \param size - display size. Acceptable values W128xH32 or W128xH64
         SSD1306(i2c_inst *i2CInst, uint16_t Address, Size size);
 
+#if __cplusplus >= 201103L
+	/// \brief SSD1306 constructor with implied standard address.
+        /// \param i2CInst - i2c instance. Either i2c0 or i2c1
+        /// \param size - display size. Acceptable values W128xH32 or W128xH64
+        SSD1306(i2c_inst *i2CInst, Size size) : SSD1306(i2CInst, size == Size::W128xH32 ? 0x3C : 0x3D, size) { }
```

Comment:
I this might not be a valid assumption. Most of my 128x64's are 0x3C, one is not, dependant on brand. It might be confusing to users if they see this assuming it will choose a sane default for them when such a default is entirely model dependant.